CONTRIBUTING.md

```diff
@@ -47,44 +48,47 @@ the expected behavior that doesn't occur.
 
 To suggest an enhancement, open an
 [issue](https://github.com/nginx/unit/issues/new) on GitHub with the label
-`enhancement`.  Please do this before implementing a new feature to discuss the
-feature first.
+`enhancement`.  Please do this before implementing a new feature to discuss
+the feature first.
 
 
 ### Open a Pull Request
 
-Before submitting a PR, please read the NGINX Unit code guidelines to know more
-about coding conventions and benchmarks.  Fork the repo, create a branch, and
-submit a PR when your changes are tested and ready for review.  Again, if you'd
-like to implement a new feature, please consider creating a feature request
-issue first to start a discussion about the feature.
+Before submitting a PR, please read the NGINX Unit code guidelines to know
+more about coding conventions and benchmarks.  Fork the repo, create a branch,
+and submit a PR when your changes are tested and ready for review.  Again, if
+you'd like to implement a new feature, please consider creating a feature
+request issue first to start a discussion about the feature.
 
 
 ## Git Style Guide
 
-- Keep a clean, concise and meaningful `git commit` history on your branch,
-  rebasing locally and squashing before submitting a PR
+- Create atomic commits.  A commit should do just one thing, I.e you
+  shouldn't mix whitespace changes with actual code changes.  Do the
```

Comment:
I woulc argue that whitespace changes are also actual code changes. This makes it feel like whitespace changes are somehow lesser than "real" code changes, which isn't something we should allude to. Maybe functional changes?
```suggestion
  shouldn't mix whitespace changes with functional code changes.  Do the
```

Reply:
It of course depends on the language...

In C, whitespace is not significant, whereas in Python it is... seeing as we're a mostly C project, I'd say the above holds, but yes we do have some other language code where whitespace _matters_ e.g Python, so I'll change as suggested.